.gitpod.yml

```diff
@@ -28,11 +28,11 @@ image:
 
 tasks:
   - init: |
-      npm install
+      pnpm install
```

Comment:
I tried using Gitpod to run the Cypress tests, but Gitpod doesn't properly install Cypress.

I ran `pnpm install` on the Gitpod instance, but that did not get Cypress working.

Reply:
I'll take a look!

Reply:
Fixed! Cypress is working correctly on GitPod now. Can you confirm is ok?